src/dev-app/focus-trap/focus-trap-demo.ts

```diff
@@ -6,14 +6,18 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {FocusTrap, FocusTrapFactory} from '@angular/cdk/a11y';
+import {CdkTrapFocus} from '@angular/cdk/a11y';
 import {
   AfterViewInit,
   Component,
   ElementRef,
   ViewChild,
-  ViewEncapsulation} from '@angular/core';
+  ViewEncapsulation,
+  ViewChildren,
+  QueryList,
+} from '@angular/core';
 import {MatDialog} from '@angular/material/dialog';
+import {_supportsShadowDom} from '@angular/cdk/platform';
```

Comment:
Out of curiosity: did we ever consider just publishing this without underscore?

Reply:
I don't remember if we ever discussed making it public.